libs/common/src/test-helpers/mock-data/utilisation-report.entity.mock-builder.ts

```diff
@@ -1,13 +1,13 @@
-import { REQUEST_PLATFORM_TYPE } from '../../constants';
+import { REQUEST_PLATFORM_TYPE, UTILISATION_REPORT_RECONCILIATION_STATUS } from '../../constants';
 import { AzureFileInfoEntity, DbRequestSource, FeeRecordEntity, UtilisationReportEntity, ReportPeriodPartialEntity } from '../../sql-db-entities';
 import { UtilisationReportReconciliationStatus } from '../../types';
 import { MOCK_AZURE_FILE_INFO } from './azure-file-info.mock';
 
 export class UtilisationReportEntityMockBuilder<ReportStatus extends UtilisationReportReconciliationStatus> {
   private readonly report: UtilisationReportEntity;
 
-  private constructor(report: UtilisationReportEntity) {
-    this.report = report;
+  public constructor(report?: UtilisationReportEntity) {
+    this.report = report ?? UtilisationReportEntityMockBuilder.forStatus(UTILISATION_REPORT_RECONCILIATION_STATUS.RECONCILIATION_IN_PROGRESS).report;
```

Comment:
this line is pretty long.  Could either RECONCILIATION_IN_PROGRESS be dereferenced above or put in a const

Reply:
This will be covered by other PR I have open, would like to merge this first please